card/mod/01_core/set/all/name.rb

```diff
@@ -275,28 +289,25 @@ def suspend_name name
   self.update_referencers = false if update_referencers == 'false'
   Card::Reference.update_on_rename self, name, self.update_referencers
 
-  deps = dependents
-  # warn "-------------------#{name_was}---- CASCADE #{self.name} -> deps: " \
-  #      " #{deps.map(&:name)*', '} -----------------------"
-
-  @dependents = nil # reset
+  des = descendants
+  @descendant_names = nil # reset
 
-  deps.each do |dep|
+  des.each do |de|
     # here we specifically want NOT to invoke recursive cascades on these
     # cards, have to go this low level to avoid callbacks.
-    Rails.logger.info "cascading name: #{dep.name}"
-    Card.expire dep.name # old name
-    newname = dep.cardname.replace_part name_was, name
-    Card.where(id: dep.id).update_all name: newname.to_s, key: newname.key
-    Card::Reference.update_on_rename dep, newname, update_referencers
+    Rails.logger.info "cascading name: #{de.name}"
+    Card.expire de.name # old name
+    newname = de.cardname.replace_part name_was, name
+    Card.where(id: de.id).update_all name: newname.to_s, key: newname.key
+    Card::Reference.update_on_rename de, newname, update_referencers
     Card.expire newname
   end
-  execute_referencers_update(deps) if update_referencers
+  execute_referencers_update(des) if update_referencers
 end
 
-def execute_referencers_update dependents
+def execute_referencers_update descendants
```

Comment:
Assignment Branch Condition size for execute_referencers_update is too high. [19.13/15]